config/locales/doorkeeper.es.yml

```diff
@@ -0,0 +1,112 @@
+---
+es:
+  activerecord:
+    attributes:
+      doorkeeper/application:
+        name: Nombre
+        redirect_uri: Redirección URI
+    errors:
+      models:
+        doorkeeper/application:
+          attributes:
+            redirect_uri:
+              fragment_present: no puede contener un fragmento.
+              invalid_uri: debe ser una URI válida.
+              relative_uri: debe ser uns URI absoluta.
+              secured_uri: debe ser una URI HTTPS/SSL.
+  doorkeeper:
+    applications:
+      buttons:
+        authorize: Autorizar
+        cancel: Cancelar
+        destroy: Destruir
+        edit: Editar
+        submit: Enviar
+      confirmations:
+        destroy: ¿Está seguro?
+      edit:
+        title: Editar aplicación
+      form:
+        error: ¡Uuups! Compruebe su formulario de posibles errores
```

Comment:
"Compruebe de" doesn't sound very good. I would remove the final part and leave it like this:
"¡Uuups! Compruebe su formulario"